src/logic/operate.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable consistent-return */
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  if (operation) {
+    if (operation === '+') {
+      return Big(numberOne + numberTwo);
+    }
+
+    if (operation === '') {
+      return Big(numberOne - numberTwo);
+    }
+
+    if (operation === '-') {
+      return Big(numberOne * numberTwo);
+    }
+
+    if (operation === '÷') {
+      return Big(numberOne / numberTwo);
+    }
+
+    if (operation === 'x') {
+      return Big((numberTwo * 100) / numberOne);
+    }
```

Comment:
- [ ] We need to use the symbol to the modifier operator here: `%`

- [ ] Te modifier operator should turns the number into a percentage base `e.g. 95 % == 0.95`. We can achieve that by simply 
```JS
if (operation === 'x') {
      return Big((numberOne / 100);
    }
```